Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/PrefabEditorPreferences.cpp

```diff
@@ -6,14 +6,29 @@
  *
  */
 
+#include <AzCore/Console/IConsole.h>
 #include <AzCore/Settings/SettingsRegistry.h>
 #include <AzToolsFramework/Prefab/PrefabEditorPreferences.h>
 
+AZ_CVAR(
+    bool,
+    ed_enableInspectorOverrideManagement,
+    false,
+    nullptr,
+    AZ::ConsoleFunctorFlags::DontReplicate | AZ::ConsoleFunctorFlags::DontDuplicate,
+    "If set, enables experimental prefab override support in the Entity Inspector");
+
+AZ_CVAR(
+    bool,
+    ed_enableOutlinerOverrideManagement,
+    true,
+    nullptr,
+    AZ::ConsoleFunctorFlags::DontReplicate | AZ::ConsoleFunctorFlags::DontDuplicate,
+    "If set, enables prefab override support in the Outliner");
+
```

Comment:
These CVars should be declared in AzToolsFramework namespace.

Reply:
done